Makefile

```diff
@@ -1,21 +1,72 @@
-default: build run 
+SHELL := /bin/bash
+DATABASE_VOLUME := food
 
+# List of all targets which are meant to be run by the user (should have a description)
+ALL_TARGETS := $(shell sed -n 's/^## //p' ${MAKEFILE_LIST} | awk -F ':' '{print $$1}')
```

Comment:
```
ALL_TARGETS := $(MAKE) -pRrq -f $(firstword $(MAKEFILE_LIST)) : 2>/dev/null | awk -v RS= -F: '/(^|\n)
```

Reply:
I believe the reference is [this](https://stackoverflow.com/questions/4219255/how-do-you-get-the-list-of-targets-in-a-makefile).
The complete command is actually longer (including the thing that looks like a comment). And it doesnt work

Reply:
My bad, didn't copy the full thing. Works on my end though, previous one didn't work.

Reply:
```
ALL_TARGETS := $(MAKE) -pRrq -f $(firstword $(MAKEFILE_LIST)) : 2>/dev/null | awk -v RS= -F: '/^# ?File/,/^# Finished Make data/ {if ($$0 !~ "^[#@%]") {print substr($$0, 3)}}'
```

This should be the command. Try if it works.

Reply:
The actual solution is so simple that i wanna crash my head somewhere